components/admin_console/admin_definition.jsx

```diff
@@ -2684,6 +2684,16 @@ export default {
                             help_text: 'admin.support.termsOfServiceTextHelp',
                             help_text_default: 'Text that will appear in your custom Terms of Service. Supports Markdown-formatted text.',
                         },
+                        {
+                            type: Constants.SettingsTypes.TYPE_NUMBER,
+                            key: 'SupportSettings.ReAcceptancePeriod',
```

Comment:
If my proposal here is approved, the key for this setting may change: https://github.com/mattermost/mattermost-server/pull/9731#pullrequestreview-171532770

Reply:
Updated this flag